### PR TITLE
Add missing open source community health files

### PR DESCRIPTION
**Pull request by jzrilic**
Adding missing open source community health files:

- issue templates for bug report, enhancement request and feature request
- pull request template
- contributing guidelines document

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/01_bugReport.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  Feel free to remove any section in the template that is not relevant or needed for your report.
+  However, keep in mind that filling out those sections can significantly reduce resolution duration.
+-->
+
+## Basic information
+* nrich version:
+  <!-- released version or snapshot version -->
+  <!-- for snapshot version include main branch commit SHA (i.e. as given from "git rev-parse HEAD" command) -->
+* Gradle version info:
```
Comment (agrancaric) [inline .github/ISSUE_TEMPLATE/01_bugReport.md:21]: I am only commenting here but it applies to other files as well. I am not sure that we need a commit SHA or will the users of the library be able to find it easily so I would remove that part. As for Gradle version info I would rename that section to Additional information (not required but might be helpful) and change it to include Java version, Maven or Gradle version and OS.

**Review by agrancaric — CHANGES_REQUESTED** [review]
Left a comment about changes needed.

File: .github/ISSUE_TEMPLATE/01_bugReport.md (hunk 1 above)
Comment (jzrilic) [reply .github/ISSUE_TEMPLATE/01_bugReport.md:21]: Maybe we can make commit SHA optional (e.g. `for snapshot version include main branch commit SHA if possible`)?

Related to the Gradle version, is it ok to introduce a new section named `Additional information` that is on the same level as `Basic information`? Not sure if there is a need to force some kind of form like in `Basic information` section or just make it plain text where user can describe details about Maven/Gradle, Java ond OS version?

Comment (agrancaric) [reply .github/ISSUE_TEMPLATE/01_bugReport.md:21]: I mean for me that SHA is not something I think it is that important so I would leave it out (for now), as for Basic and Additional information I leave it up to you how to format it, for me everything except nrich version is additional information that can help but is not strictly required.

Comment (jzrilic) [reply .github/ISSUE_TEMPLATE/01_bugReport.md]: Please check 0313ff5f3e5bf0600a4afc4d2b7e2ae688cbede4 and if it's ok.

**Review by agrancaric — APPROVED** [review]

**Review by jsajlovic — APPROVED** [review]

**Pull request merged**